.github/CODEOWNERS

```diff
@@ -178,6 +178,9 @@
 # ServiceLabel: %Cloud Shell
 # ServiceOwners:                                                   @maertendMSFT
 
+# ServiceLabel: %Code Signing
+# ServiceOwners:                                                   @jaxelr
```

Comment:
```suggestion
#PRLabel %Code Signing
/sdk/trustedsigning/                                               @jaxelr

# ServiceLabel: %Code Signing
# ServiceOwners:                                                   @jaxelr
```

Reply:
You need entries for both pull requests and support issues.   In this case, I'm assuming that you're the correct contact for both.

Reply:
Yes, I'll be the initial contact, will more than likely subscribe more individuals in the future. Thanks this has been fixed on the latest commit.